training_config/coref_bert_lstm.jsonnet

```diff
@@ -0,0 +1,103 @@
+// Configuration for a coreference resolution model based on:
```

Comment:
Can't remember if I've mentioned it before, but one nice feature of jsonnet is that you can de-duplicate across files. See, e.g., https://github.com/allenai/allennlp/blob/master/allennlp/tests/fixtures/language_model/experiment_unidirectional_unsampled.jsonnet. But that's just an FYI. Given that we're going to get rid of the old coref.jsonnet soon anyway, I wouldn't bother for this PR.